### PR TITLE
qa/cephfs: switch to python3 for centos stream 9

### PR DESCRIPTION
**Pull request by lxbsz**
Fixes: https://tracker.ceph.com/issues/62277

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: qa/tasks/cephfs/xfstests_dev.py [hunk 1]
```diff
@@ -202,6 +202,8 @@ def install_deps(self):
             if self.install_xfsprogs:
                 if distro == 'centosstream' and major_ver_num == 8:
                     deps += ['--enablerepo=powertools']
+                if distro == 'centosstream' and major_ver_num == 9:
```
Comment (vshankar) [inline qa/tasks/cephfs/xfstests_dev.py:205]: Can't we just switch to `python3` in the deps list above? The ubuntu distro case seems to be correctly handled below.

Comment (dparmar18) [reply qa/tasks/cephfs/xfstests_dev.py:205]: +1, anyways py2 is deprecated. py3 is what we should default to

Comment (lxbsz) [reply qa/tasks/cephfs/xfstests_dev.py:205]: Okay, sounds reasonable.

**Review by dparmar18 — APPROVED** [review]

**Review by vshankar — APPROVED** [review]

**Comment by vshankar** [comment]
jenkins test make check

**Comment by vshankar** [comment]
https://pulpito.ceph.com/vshankar-2023-08-09_05:46:29-fs-wip-vshankar-testing-20230809.035933-testing-default-smithi/

**Comment by vshankar** [comment]
Test update: Tests passed, preparing run wiki - will merge soon.

**Review by vshankar — APPROVED** [review]
https://tracker.ceph.com/projects/cephfs/wiki/Main#22-Aug-2023

**Pull request merged**